### PR TITLE
Log Analytics new parameters Group By ApplicationId and UserAgent

### PR DESCRIPTION
**Pull request by Sandido**
Work for item https://msazure.visualstudio.com/One/_workitems/edit/8868208

PS Design Review: Azure/azure-powershell-cmdlet-review-pr#794

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Compute/Compute.Test/ScenarioTests/LogAnalyticTests.ps1 [hunk 1]
```diff
@@ -43,6 +43,67 @@ function Test-ExportLogAnalyticRequestRateByIntervalNegative
...
+        $stotype = 'Standard_GRS';
+        $container = "contain";
+        New-AzStorageAccount -ResourceGroupName $rgname -Name $stoname -Location $loc -Type $stotype;
+        $key = Get-AzStorageAccountKey -ResourceGroupName $rgname -Name $stoname;
+        Assert-NotNull $key;
+
+        $from = Get-Date -Year 2021 -Month 1 -Day 4 -Hour 9;
+        $to = Get-Date -Year 2021 -Month 1 -Day 5 -Hour 9;
+
+        $context = New-AzStorageContext -StorageAccountName $stoname -StorageAccountKey $key.Value[0];
```
Comment (msJinLei) [inline src/Compute/Compute.Test/ScenarioTests/LogAnalyticTests.ps1:70]: Please check the way your module depends on Az.Storage to see whey New-AzStorageContext  cannot be recognized.

**Review by msJinLei — CHANGES_REQUESTED** [review]
@Sandido Please fix the failed test cases

File: src/Compute/Compute.Test/ScenarioTests/LogAnalyticTests.ps1 (hunk 1 above)
Comment (Sandido) [reply src/Compute/Compute.Test/ScenarioTests/LogAnalyticTests.ps1:70]: @msJinLei , this build error is not present on my local VS.

Comment (Sandido) [reply src/Compute/Compute.Test/ScenarioTests/LogAnalyticTests.ps1:70]: Could this error be due to the CredScan check failing instead?

**Comment by msJinLei** [comment]
@Sandido 
Compute test module uses storage via cmdlets defined in AzureRM.Storage.ps1.  New-AzStorageContext cannot be recognized because it is not defined in AzureRM.Storage.ps1.

**Review by msJinLei — APPROVED** [review]

**Pull request merged**